maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -424,8 +423,15 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         waitForAppToSettle()
     }
 
-    fun openLink(link: String) {
-        driver.openLink(link)
+    fun openLink(link: String, appId: String?, autoVerify: Boolean) {
+        LOGGER.info("Opening link $link for app: $appId with autoVerify config as $autoVerify")
+
+        driver.openLink(link, appId, autoVerify)
+        waitForAppToSettle()
+    }
+
+    fun openBrowser(link: String) {
+        driver.openBrowser(link)
```

Comment:
nit: add a log message, similarly to `openLink`

Reply:
Updated to add browser attribute